syntax/ps1.vim

```diff
@@ -53,35 +55,47 @@ syn match ps1Constant +\$^+
 " Keywords reserved for future use
 syn keyword ps1Keyword class define from using var
 
-" Functions and Cmdlets
-syn match ps1Cmdlet /\w\+-\w\+/
```

Comment:
As a further excercise, I went through the stock Vim syntax files -- especially C, C++, C#, Pascal, PHP, and Java -- to get a sense of the difference between how `Statement` and `Function` are used.

`Statement` is used mainly for language keywords like `return`, `break`, `continue`, etc. and occasionally (e.g. VB) for invocation of "builtin" procedures that don't return values.

`Function` is used mainly for function declarations, and occasionally (e.g. PHP) for invocation of "builtin" functions that return values.

My improvements distinguish between function declaration and invocation.  Declarations are mapped to `Function` which, given the above, seems to be correct.  Invocations are also mapped to `Function` which, given the above, seems to be *more* correct than `Statement`.